app-admin/etcd-wrapper/files/etcd-wrapper

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -e
+
+USAGE="
+Usage: $0 [etcd flags]
+
+Wrapper for launching etcd via rkt.
+
+The version of etcd to be used can be set through ETCD_VERSION. The version
+must be set to one of the image tags published here:
+
+  https://quay.io/repository/coreos/etcd?tab=tags
+
+Alternatively, the keywords 'alpha', 'beta', and 'stable' may be used to specify
+the version. If no version is provided, the CoreOS update group is used.
+"
+
+case ${1} in
+	-h | --help)
+		echo "${USAGE}"
+		exit 0
+		;;
+esac
+
+source /usr/share/coreos/update.conf
+source /etc/coreos/update.conf
+
+ETCD_VERSION="${ETCD_VERSION:-${GROUP:-stable}}"
+ETCD_ACI="${ETCD_ACI:-coreos.com/etcd}"
+
+case ${ETCD_VERSION} in
```

Comment:
The sourced files are just providing the default etcd "channel" based on the one use for coreos itself, they aren't shell scripts nor should they be have etcd info, they are configuration for update_engine.

Also, there isn't a place where we can set etcd version that will change based on the coreos channel, if it lived in config in /etc it wouldn't update during upgrades. If it lived in /usr it would have to be the same for all channels since /usr does not change between channels. So presuming this is behavior we want it has to be logic in code.

BUT... that is assuming this is something we want in the first place. Now that I think about it I'd argue that this is something we really do NOT want as it is contrary to the existing pattern of versions being promoted verbatim between alpha/beta/stable. Things get surprising when versions change like this, dropping the kublet between beta/stable has been bad enough.

So if we specify a version it should be a single version, but if it is a single version then it doesn't really need to be a container in the first place. Instead can we go with specifying no version like we did for kublet-wrapper? If we want to do alpha/beta/stable channels for etcd they should be version aliases/tags in quay.io or whatever. Doing that also means that etcd updates would not be bound to coreos updates which is probably reasonable if you want to promote etcd channels differently than coreos channels as the following code tries to do.